.ci/install_nemu.sh

```diff
@@ -20,6 +20,8 @@ latest_build_url="http://jenkins.katacontainers.io/job/nemu-nightly-${arch}/last
 nemu_repo=$(get_version "assets.hypervisor.nemu.url")
 nemu_version=$(get_version "assets.hypervisor.nemu.version")
 NEMU_TAR="kata-nemu-static.tar.gz"
+KATA_TESTS_CACHEDIR="/tmp/cache"
```

Comment:
This is defined in `.ci/ci_cache_components.sh`. We need to avoid redefining a variable with the same value multiple times as if they diverge... Can't you define the variable once and `source` that file into the scripts using it. I think I mentioned that in a previous round of reviews?